package.json

```diff
@@ -23,18 +23,18 @@
   },
   "homepage": "https://github.com/digitalbazaar/bedrock-kms-http",
   "dependencies": {
-    "@digitalbazaar/ezcap-express": "^2.0.0",
+    "@digitalbazaar/ezcap-express": "digitalbazaar/ezcap-express#v4.x",
```

Comment:
This got rolled into 3.x https://github.com/digitalbazaar/ezcap-express/blob/main/CHANGELOG.md#301---2021-04-06